plugins/entando-plugin-jpversioning/src/main/java/org/entando/entando/plugins/jpversioning/web/contentsversioning/validator/ContentVersioningValidator.java

```diff
@@ -36,4 +37,16 @@ public boolean contentVersioningExist(String contentId){
         return validationResult;
     }
 
+    public boolean checkContentIdForVersion(String contentId, Long versionId) {
+        boolean validationResult = false;
+        try {
+            final ContentVersion version = versioningManager.getVersion(versionId);
+            if (version.getContentId().equals(contentId)){
+                return true;
+            }
+        } catch (ApsSystemException e) {
+            e.printStackTrace();
```

Comment:
Can you substitute this console output with a right log?

Reply:
Done